### PR TITLE
Fix chart lines loading

### PR DESCRIPTION
**Pull request by ptbrowne**
When creating a new dataset and selecting lines chart, the chart would stay blocked in loading mode. This was due to the `fetching` attribute of the OperationResult staying to true when a query was refetched while it was already in cache. Updating urql fixes the problem 🎉 

Fixes: https://github.com/visualize-admin/visualization-tool/issues/211

Two minor commits that I was doing at the time of the fix:

- docs: Rename dataset search to dataset browse
- refactor: Extract and use set instead of array to check for inclusivity

### CODE REVIEW

File: app/configurator/components/dataset-browse.tsx [hunk 1]
```diff
@@ -1,28 +1,30 @@
 import { Maybe } from "@graphql-tools/utils/types";
 import { Plural, t, Trans } from "@lingui/macro";
 import { mapValues, pickBy, sortBy } from "lodash";
+import Link from "next/link";
+import { Router, useRouter } from "next/router";
+import qs from "qs";
```
Comment (herrstucki) [inline app/configurator/components/dataset-browse.tsx:6]: nit: These imports got added but I don't see them used?

**Review by herrstucki — APPROVED** [review]

**Comment by herrstucki** [comment]
Ah, I don't like automerge 😂 @ptbrowne can you make sure to check the imports on main?

**Comment by ptbrowne** [comment]
They are imported, and we now have an eslint rule for unused imports ;)

**Comment by herrstucki** [comment]
My point was, they should not be imported? Maybe I saw wrong 😅

**Comment by ptbrowne** [comment]
Oups sorry, I meant "they are used" (I checked). The eslint rule would error if they were not used ;)

**Comment by herrstucki** [comment]
Ah nevermind … just a quick note: you don't need to use qs.stringify, as you can just pass the query as object to `router.push()`. Just highlighting this because this would add another package to the bundle that (hopefully) is not used elsewhere and duplicates functionality built into next.js.

**Pull request merged**